Lib/test/test_heapq.py

```diff
@@ -236,6 +300,20 @@ def test_heapsort(self):
             heap_sorted = [self.module.heappop(heap) for i in range(size)]
             self.assertEqual(heap_sorted, sorted(data))
 
+    def test_heapsort_max(self):
+        for trial in range(100):
+            size = random.randrange(50)
+            data = [random.randrange(25) for i in range(size)]
+            if trial & 1:     # Half of the time, use heapify
```

Comment:
```suggestion
            if trial & 1:     # Half the time, use heapify
```

and should it also not be heapify_max

Reply:
Both are fine, according to https://english.stackexchange.com/questions/217600/should-i-use-half-the-time-or-half-of-the-time
So I'd rather match `test_heapsort`.

But being explicit in the comments is an improvement.

```suggestion
            if trial & 1:     # Half of the time, use heapify_max
```